sdk/metrics/src/main/java/io/opentelemetry/sdk/metrics/internal/view/ExplicitBucketHistogramAggregation.java

```diff
@@ -24,27 +24,35 @@
  * <p>This class is internal and is hence not for public use. Its APIs are unstable and can change
  * at any time.
  */
-public final class ExplicitBucketHistogramAggregation implements Aggregation, AggregatorFactory {
+public final class ExplicitBucketHistogramAggregation implements AggregationExtension {
 
   private static final Aggregation DEFAULT =
       new ExplicitBucketHistogramAggregation(
-          ExplicitBucketHistogramUtils.DEFAULT_HISTOGRAM_BUCKET_BOUNDARIES);
+          ExplicitBucketHistogramUtils.DEFAULT_HISTOGRAM_BUCKET_BOUNDARIES,
+          ExemplarReservoirFactory.histogramBucket(
+              Clock.getDefault(),
+              ExplicitBucketHistogramUtils.DEFAULT_HISTOGRAM_BUCKET_BOUNDARIES));
 
   public static Aggregation getDefault() {
     return DEFAULT;
   }
 
   public static Aggregation create(List<Double> bucketBoundaries) {
-    return new ExplicitBucketHistogramAggregation(bucketBoundaries);
+    return new ExplicitBucketHistogramAggregation(
+        bucketBoundaries,
+        ExemplarReservoirFactory.histogramBucket(Clock.getDefault(), bucketBoundaries));
```

Comment:
I realized that we need to use fixed size by default if `bucketBoundaries.size() == 0`.